src/modules/auth/views/Login.vue

```diff
@@ -0,0 +1,92 @@
+<script>
+import { ref } from "vue";
+import { getAuth, signInWithEmailAndPassword, GoogleAuthProvider, signInWithPopup, } from "firebase/auth";
+import { RouterLink, useRouter } from "vue-router";
+
+const email = ref("");
+const password = ref("");
+const errMsg = ref(); //msg para erro
```

Comment:
o nome da variável já está explicativo ;)
```suggestion
const errMsg = ref(); 
```